DylibBinder/XmlGenerator.cs

```diff
@@ -0,0 +1,277 @@
+﻿using System;
+using System.Linq;
+using System.Xml;
+
+namespace DylibBinder {
+	sealed internal class XmlGenerator : IDisposable {
+		public XmlGenerator (DBTopLevel dBTopLevel, string outputPath)
+		{
+			writer = CreateWriter (outputPath);
+			WriteIntro (dBTopLevel);
+			WriteTypeDeclarations (dBTopLevel);
+			CloseWriter ();
+		}
+
+		XmlWriter writer;
+
+		XmlWriter CreateWriter (string outputPath)
+		{
+			XmlWriterSettings settings = new XmlWriterSettings ();
+			settings.Indent = true;
+			settings.IndentChars = "   ";
```

Comment:
"\t" maybe?

Reply:
I decided to use 3 spaces to copy the indentation that the reflector uses